doc/classes/Mesh.xml

```diff
@@ -40,6 +40,13 @@
 				Generate a [TriangleMesh] from the mesh.
 			</description>
 		</method>
+		<method name="get_aabb" qualifiers="const">
+			<return type="AABB">
+			</return>
+			<description>
+				Returns the smallest AABB enclosing this Mesh.
```

Comment:
Should either be "mesh" (lower case, English word) or "[Mesh]" (class name, with hyperlink - or here, code markup, as it's the class being described).

Reply:
I'd also add a "[b]Note:[/b]" that the method is only implemented for [ArrayMesh] and [PrimitiveMesh].